tests/testthat/test-5makePhyloseqFromTreeSummarizedExperiment.R

```diff
@@ -0,0 +1,47 @@
+context("makePhyloseqFromTreeSummarizedExperiment")
+
+test_that("makePhyloseqFromTreeSummarizedExperiment", {
+
+    skip_if_not_installed("phyloseq")
+
+    # TSE object
+    data(GlobalPatterns)
+    tse <- GlobalPatterns
+
+    phy <- makePhyloseqFromTreeSummarizedExperiment(GlobalPatterns)
+
+    # Test that assay is in otu_table
+    testthat::expect_equal(as.data.frame(phyloseq::otu_table(phy)@.Data), as.data.frame(assays(tse)$counts))
```

Comment:
the `testthat::` prefix is not necassary. If you want to test this locally either load `testthat` in your session or use `devtools::test(filter="5makePhyloseq")`

Reply:
Removed testthat::